haralyzer/assets.py

```diff
@@ -286,6 +290,8 @@ def _get_asset_load(self, asset_type):
         elif asset_type == 'content':
             return self.pageTimings['onContentLoad']
         elif asset_type == 'page':
+            if self.page_id == 'unknown':
+                return None
```

Comment:
Returning `None` instead of `0` here could potentially be a breaking change for current users. The problem is that (at least personally) I would interpret `0` to mean "0 milliseconds" not "it does not exist". Need to think about this a bit to make sure it will not be a problem for users currently expecting a `0` in such situations.

Reply:
EDIT: This would only affect people with "unknown" pages, which means that they would have already stumbled upon the error that this PR is intended to fix. As such, we should be good to go here.